[2014-06-30 09:21:38] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'syntax error, unexpected '$new_number' (T_VARIABLE)' in /opt/lampp/htdocs/noyawa/app/controllers/ClientController.php:63 Stack trace: #0 [internal function]: Illuminate\Exception\Handler->handleShutdown() #1 {main} [] []
[2014-06-30 09:22:53] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'syntax error, unexpected '$input' (T_VARIABLE)' in /opt/lampp/htdocs/noyawa/app/controllers/ClientController.php:63 Stack trace: #0 [internal function]: Illuminate\Exception\Handler->handleShutdown() #1 {main} [] []
[2014-06-30 09:23:51] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'syntax error, unexpected 'Completed' (T_STRING)' in /opt/lampp/htdocs/noyawa/app/controllers/ClientController.php:63 Stack trace: #0 [internal function]: Illuminate\Exception\Handler->handleShutdown() #1 {main} [] []
[2014-06-30 09:24:57] log.INFO: Accessing postClient @ ClientController [] []
[2014-06-30 09:24:57] log.ERROR: exception 'ErrorException' with message 'Undefined variable: client' in /opt/lampp/htdocs/noyawa/app/controllers/ClientController.php:86 Stack trace: #0 /opt/lampp/htdocs/noyawa/app/controllers/ClientController.php(86): Illuminate\Exception\Handler->handleError(8, 'Undefined varia...', '/opt/lampp/htdo...', 86, Array) #1 [internal function]: ClientController->postClient() #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('postClient', Array) #4 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'postClient', Array) #5 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #6 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #7 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #8 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #9 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #10 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #11 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #12 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #13 {main} [] []
[2014-06-30 09:26:03] log.INFO: Accessing postClient @ ClientController [] []
[2014-06-30 09:38:25] log.INFO: Accessing postClient @ ClientController [] []
[2014-06-30 09:39:07] log.INFO: Accessing postClient @ ClientController [] []
[2014-06-30 09:39:32] log.INFO: Accessing postClient @ ClientController [] []
[2014-06-30 09:51:07] log.INFO: Accessing postClient @ ClientController [] []
[2014-06-30 09:52:02] log.INFO: Accessing postClient @ ClientController [] []
[2014-06-30 10:11:06] log.INFO: Accessing postClient @ ClientController [] []
[2014-06-30 11:24:21] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-06-30 11:24:32] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-06-30 11:24:47] log.INFO: Uploading excel file -> test.xlsx [] []
[2014-06-30 11:24:47] log.INFO: File uploaded -> test.xlsx :: Processing started... [] []
[2014-06-30 11:24:47] log.INFO: File is readable -> test.xlsx [] []
[2014-06-30 11:24:48] log.INFO: File has 10 records [] []
[2014-06-30 11:24:49] log.INFO: Client [233246150548] saved! [] []
[2014-06-30 11:24:49] log.INFO: Client [233546941942] saved! [] []
[2014-06-30 11:24:49] log.INFO: Client [233249506770] saved! [] []
[2014-06-30 11:24:49] log.INFO: Client [233249647522] saved! [] []
[2014-06-30 11:24:49] log.INFO: Client [233240440316] saved! [] []
[2014-06-30 11:24:49] log.INFO: Client [233543496795] saved! [] []
[2014-06-30 11:24:49] log.INFO: Client [233549083318] saved! [] []
[2014-06-30 11:24:49] log.INFO: Client [233541728956] saved! [] []
[2014-06-30 11:24:49] log.INFO: Client [233546295110] saved! [] []
[2014-06-30 11:24:49] log.INFO: Processing of file completed [] []
[2014-06-30 12:26:13] log.ERROR: exception 'PDOException' with message 'SQLSTATE[42S22]: Column not found: 1054 Unknown column 'excel_uploads.file_name ' in 'field list'' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Database/Connection.php:272 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Database/Connection.php(272): PDO->prepare('select `excel_u...') #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Database/Connection.php(521): Illuminate\Database\Connection->Illuminate\Database\{closure}(Object(Illuminate\Database\MySqlConnection), 'select `excel_u...', Array) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Database/Connection.php(277): Illuminate\Database\Connection->run('select `excel_u...', Array, Object(Closure)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(1028): Illuminate\Database\Connection->select('select `excel_u...', Array) #4 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(1018): Illuminate\Database\Query\Builder->runSelect() #5 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(1005): Illuminate\Database\Query\Builder->getFresh(Array) #6 /opt/lampp/htdocs/noyawa/vendor/bllim/datatables/src/Bllim/Datatables/Datatables.php(89): Illuminate\Database\Query\Builder->get() #7 /opt/lampp/htdocs/noyawa/vendor/bllim/datatables/src/Bllim/Datatables/Datatables.php(68): Bllim\Datatables\Datatables->get_result() #8 /opt/lampp/htdocs/noyawa/app/controllers/UploadController.php(19): Bllim\Datatables\Datatables->make() #9 [internal function]: UploadController->viewData() #10 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #11 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('viewData', Array) #12 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'viewData', Array) #13 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #14 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #15 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #16 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #17 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #18 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #19 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #20 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #21 {main} Next exception 'Exception' with message 'SQLSTATE[42S22]: Column not found: 1054 Unknown column 'excel_uploads.file_name ' in 'field list' (SQL: select `excel_uploads`.`id` as `id`, `excel_uploads`.`file_name `, `excel_uploads`.`file_extension`, `excel_uploads`.`number_of_records`, `excel_uploads`.`status`, `excel_uploads`.`created_at` from `excel_uploads`) (Bindings: array ( ))' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Database/Connection.php:556 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Database/Connection.php(529): Illuminate\Database\Connection->handleQueryException(Object(PDOException), 'select `excel_u...', Array) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Database/Connection.php(277): Illuminate\Database\Connection->run('select `excel_u...', Array, Object(Closure)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(1028): Illuminate\Database\Connection->select('select `excel_u...', Array) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(1018): Illuminate\Database\Query\Builder->runSelect() #4 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(1005): Illuminate\Database\Query\Builder->getFresh(Array) #5 /opt/lampp/htdocs/noyawa/vendor/bllim/datatables/src/Bllim/Datatables/Datatables.php(89): Illuminate\Database\Query\Builder->get() #6 /opt/lampp/htdocs/noyawa/vendor/bllim/datatables/src/Bllim/Datatables/Datatables.php(68): Bllim\Datatables\Datatables->get_result() #7 /opt/lampp/htdocs/noyawa/app/controllers/UploadController.php(19): Bllim\Datatables\Datatables->make() #8 [internal function]: UploadController->viewData() #9 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #10 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('viewData', Array) #11 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'viewData', Array) #12 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #13 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #14 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #15 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #16 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #17 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #18 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #19 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #20 {main} [] []
[2014-06-30 12:28:05] log.ERROR: exception 'PDOException' with message 'SQLSTATE[42S22]: Column not found: 1054 Unknown column 'id ' in 'field list'' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Database/Connection.php:272 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Database/Connection.php(272): PDO->prepare('select `id `, `...') #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Database/Connection.php(521): Illuminate\Database\Connection->Illuminate\Database\{closure}(Object(Illuminate\Database\MySqlConnection), 'select `id `, `...', Array) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Database/Connection.php(277): Illuminate\Database\Connection->run('select `id `, `...', Array, Object(Closure)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(1028): Illuminate\Database\Connection->select('select `id `, `...', Array) #4 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(1018): Illuminate\Database\Query\Builder->runSelect() #5 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(1005): Illuminate\Database\Query\Builder->getFresh(Array) #6 /opt/lampp/htdocs/noyawa/vendor/bllim/datatables/src/Bllim/Datatables/Datatables.php(89): Illuminate\Database\Query\Builder->get() #7 /opt/lampp/htdocs/noyawa/vendor/bllim/datatables/src/Bllim/Datatables/Datatables.php(68): Bllim\Datatables\Datatables->get_result() #8 /opt/lampp/htdocs/noyawa/app/controllers/UploadController.php(19): Bllim\Datatables\Datatables->make() #9 [internal function]: UploadController->viewData() #10 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #11 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('viewData', Array) #12 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'viewData', Array) #13 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #14 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #15 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #16 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #17 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #18 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #19 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #20 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #21 {main} Next exception 'Exception' with message 'SQLSTATE[42S22]: Column not found: 1054 Unknown column 'id ' in 'field list' (SQL: select `id `, `file_name `, `file_extension`, `number_of_records`, `status`, `created_at` from `excel_uploads`) (Bindings: array ( ))' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Database/Connection.php:556 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Database/Connection.php(529): Illuminate\Database\Connection->handleQueryException(Object(PDOException), 'select `id `, `...', Array) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Database/Connection.php(277): Illuminate\Database\Connection->run('select `id `, `...', Array, Object(Closure)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(1028): Illuminate\Database\Connection->select('select `id `, `...', Array) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(1018): Illuminate\Database\Query\Builder->runSelect() #4 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(1005): Illuminate\Database\Query\Builder->getFresh(Array) #5 /opt/lampp/htdocs/noyawa/vendor/bllim/datatables/src/Bllim/Datatables/Datatables.php(89): Illuminate\Database\Query\Builder->get() #6 /opt/lampp/htdocs/noyawa/vendor/bllim/datatables/src/Bllim/Datatables/Datatables.php(68): Bllim\Datatables\Datatables->get_result() #7 /opt/lampp/htdocs/noyawa/app/controllers/UploadController.php(19): Bllim\Datatables\Datatables->make() #8 [internal function]: UploadController->viewData() #9 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #10 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('viewData', Array) #11 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'viewData', Array) #12 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #13 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #14 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #15 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #16 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #17 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #18 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #19 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #20 {main} [] []
[2014-06-30 12:30:27] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-06-30 12:30:27] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-06-30 12:32:59] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-06-30 12:32:59] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-06-30 12:49:23] log.ERROR: exception 'PDOException' with message 'SQLSTATE[42S02]: Base table or view not found: 1146 Table 'noyawa.uploads' doesn't exist' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Database/Connection.php:272 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Database/Connection.php(272): PDO->prepare('select count(*)...') #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Database/Connection.php(521): Illuminate\Database\Connection->Illuminate\Database\{closure}(Object(Illuminate\Database\MySqlConnection), 'select count(*)...', Array) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Database/Connection.php(277): Illuminate\Database\Connection->run('select count(*)...', Array, Object(Closure)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(1028): Illuminate\Database\Connection->select('select count(*)...', Array) #4 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(1018): Illuminate\Database\Query\Builder->runSelect() #5 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(1005): Illuminate\Database\Query\Builder->getFresh(Array) #6 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(1369): Illuminate\Database\Query\Builder->get(Array) #7 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(1311): Illuminate\Database\Query\Builder->aggregate('count', Array) #8 /opt/lampp/htdocs/noyawa/vendor/chumper/datatable/src/Chumper/Datatable/Engines/QueryEngine.php(109): Illuminate\Database\Query\Builder->count() #9 /opt/lampp/htdocs/noyawa/vendor/chumper/datatable/src/Chumper/Datatable/Engines/BaseEngine.php(225): Chumper\Datatable\Engines\QueryEngine->internalMake(Object(Illuminate\Support\Collection), Array) #10 /opt/lampp/htdocs/noyawa/app/controllers/UploadController.php(18): Chumper\Datatable\Engines\BaseEngine->make() #11 [internal function]: UploadController->viewData() #12 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #13 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('viewData', Array) #14 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'viewData', Array) #15 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #16 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #17 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #18 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #19 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #20 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #21 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #22 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #23 {main} Next exception 'Exception' with message 'SQLSTATE[42S02]: Base table or view not found: 1146 Table 'noyawa.uploads' doesn't exist (SQL: select count(*) as aggregate from `uploads`) (Bindings: array ( ))' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Database/Connection.php:556 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Database/Connection.php(529): Illuminate\Database\Connection->handleQueryException(Object(PDOException), 'select count(*)...', Array) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Database/Connection.php(277): Illuminate\Database\Connection->run('select count(*)...', Array, Object(Closure)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(1028): Illuminate\Database\Connection->select('select count(*)...', Array) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(1018): Illuminate\Database\Query\Builder->runSelect() #4 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(1005): Illuminate\Database\Query\Builder->getFresh(Array) #5 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(1369): Illuminate\Database\Query\Builder->get(Array) #6 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(1311): Illuminate\Database\Query\Builder->aggregate('count', Array) #7 /opt/lampp/htdocs/noyawa/vendor/chumper/datatable/src/Chumper/Datatable/Engines/QueryEngine.php(109): Illuminate\Database\Query\Builder->count() #8 /opt/lampp/htdocs/noyawa/vendor/chumper/datatable/src/Chumper/Datatable/Engines/BaseEngine.php(225): Chumper\Datatable\Engines\QueryEngine->internalMake(Object(Illuminate\Support\Collection), Array) #9 /opt/lampp/htdocs/noyawa/app/controllers/UploadController.php(18): Chumper\Datatable\Engines\BaseEngine->make() #10 [internal function]: UploadController->viewData() #11 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #12 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('viewData', Array) #13 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'viewData', Array) #14 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #15 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #16 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #17 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #18 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #19 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #20 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #21 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #22 {main} [] []
[2014-06-30 12:54:12] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-06-30 12:54:12] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-06-30 12:54:59] log.ERROR: exception 'Symfony\Component\Routing\Exception\RouteNotFoundException' with message 'Unable to generate a URL for the named route "/viewuploads" as such route does not exist.' in /opt/lampp/htdocs/noyawa/vendor/symfony/routing/Symfony/Component/Routing/Generator/UrlGenerator.php:134 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/UrlGenerator.php(192): Symfony\Component\Routing\Generator\UrlGenerator->generate('/viewuploads', Array, true) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Support/helpers.php(696): Illuminate\Routing\UrlGenerator->route('/viewuploads', Array, true) #2 /opt/lampp/htdocs/noyawa/app/storage/views/da0603b62b2d8cea6264c710a95731ac(5): route('/viewuploads') #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/View/Engines/PhpEngine.php(38): include('/opt/lampp/htdo...') #4 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/View/Engines/CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('/opt/lampp/htdo...', Array) #5 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/View/View.php(100): Illuminate\View\Engines\CompilerEngine->get('/opt/lampp/htdo...', Array) #6 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/View/View.php(81): Illuminate\View\View->getContents() #7 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Http/Response.php(70): Illuminate\View\View->render() #8 /opt/lampp/htdocs/noyawa/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View)) #9 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View)) #10 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Route.php(58): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request)) #11 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #12 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #13 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #14 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #15 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #16 {main} [] []
[2014-06-30 12:55:10] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-06-30 12:55:10] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-06-30 13:00:02] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-06-30 13:00:02] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-06-30 13:45:26] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-06-30 13:45:26] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-06-30 13:46:54] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-06-30 13:46:54] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-06-30 13:49:38] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-06-30 13:49:38] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-06-30 13:50:59] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-06-30 13:50:59] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-06-30 13:51:43] log.ERROR: exception 'ErrorException' with message 'Object of class Illuminate\Database\Query\Builder could not be converted to string' in /opt/lampp/htdocs/noyawa/vendor/monolog/monolog/src/Monolog/Logger.php:237 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/monolog/monolog/src/Monolog/Logger.php(237): Illuminate\Exception\Handler->handleError(4096, 'Object of class...', '/opt/lampp/htdo...', 237, Array) #1 /opt/lampp/htdocs/noyawa/vendor/monolog/monolog/src/Monolog/Logger.php(291): Monolog\Logger->addRecord(200, Object(Illuminate\Database\Query\Builder), Array) #2 [internal function]: Monolog\Logger->addInfo(Object(Illuminate\Database\Query\Builder)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Log/Writer.php(207): call_user_func_array(Array, Array) #4 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php(206): Illuminate\Log\Writer->__call('info', Array) #5 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php(206): Illuminate\Log\Writer->info(Object(Illuminate\Database\Query\Builder)) #6 /opt/lampp/htdocs/noyawa/app/controllers/UploadController.php(17): Illuminate\Support\Facades\Facade::__callStatic('info', Array) #7 /opt/lampp/htdocs/noyawa/app/controllers/UploadController.php(17): Illuminate\Support\Facades\Log::info(Object(Illuminate\Database\Query\Builder)) #8 [internal function]: UploadController->viewUploads() #9 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #10 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('viewUploads', Array) #11 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'viewUploads', Array) #12 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #13 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #14 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #15 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #16 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #17 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #18 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #19 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #20 {main} [] []
[2014-06-30 13:53:24] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-06-30 13:53:24] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-06-30 13:53:26] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-06-30 13:53:26] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-06-30 13:53:49] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-06-30 13:53:49] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-06-30 13:54:14] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-06-30 13:54:14] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-06-30 13:54:16] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-06-30 13:54:16] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-06-30 13:54:29] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-06-30 13:54:29] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-06-30 14:40:59] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-06-30 14:41:00] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-06-30 14:41:03] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-06-30 14:41:03] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-06-30 14:41:41] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-06-30 14:41:41] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-06-30 15:15:46] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-06-30 15:15:46] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-06-30 15:16:09] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-06-30 15:16:09] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-06-30 15:16:45] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-06-30 15:16:45] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-06-30 15:17:52] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-06-30 15:17:53] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-06-30 15:18:28] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-06-30 15:18:28] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-06-30 15:18:51] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-06-30 15:18:51] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
